concepts/admin.py

```diff
@@ -20,30 +23,49 @@ def merge_concepts(modeladmin, request, queryset):
     :param queryset:
     :return:
     """
-    
-    opts = modeladmin.model._meta
-    app_label = opts.app_label
+    #return render_to_response('admin/Sample.html')
     RESOLVED_STATUS = Concept.RESOLVED
-    resolvedConceptCount = queryset.filter(concept_state = RESOLVED_STATUS).count()
 
-    #VGNWB-121 gets called only when there is only one resolved concept
-    if resolvedConceptCount == 1:
-        resolvedConceptsList = queryset.filter(concept_state = RESOLVED_STATUS).values()
-        unResolvedConceptsList = queryset.exclude(concept_state = RESOLVED_STATUS).values()
 
-        #As there will be only one element in the list
+    if 'post' in request.POST:
```

Comment:
Not sure why ``'post'`` would be in ``request.POST``; are you trying to check whether the request is a POST request? If so, see [this snippet](https://docs.djangoproject.com/en/1.9/ref/request-response/#django.http.HttpRequest.method) for an example.